plan/analyze_test.go

```diff
@@ -66,6 +74,14 @@ func (s *testAnalyzeSuite) TestAnalyze(c *C) {
 			sql:  "select * from t1 where t1.a = 1 and t1.b <= 2",
 			best: "Index(t1.a)[[1,1]]",
 		},
+		{
+			sql:  "select * from t2 where t2.a <= 2",
+			best: "Index(t2.b)[[<nil>,+inf]]",
```

Comment:
We only analyze the index a, so the table row count is accurate(which is 8), but since the histogram for index b is pseudo, the row count is cacluated by `getPseudoRowCountByIndexRanges`, which for such a small table, will always return `tableRowCount/3`, and makes the cost small. Not sure if it is needed to fix. PTAL　@coocood @hanfei1991